concent_api/core/decorators.py

```diff
@@ -225,6 +226,11 @@ def wrapper(
             except UnsupportedProtocolVersion as exception:
                 return HttpResponse(exception.error_message, status=404)
 
+            except SCINotSynchronized:
+                json_response = JsonResponse({'error': 'SCI is currently not synchronized'}, status=502)
```

Comment:
I think we should use error code for this, as the client might want to handle this situation somehow.

Reply:
Done